corona-warn-app/cwa-app-android
# Embed country list for submission of keys screen (EXPOSUREAPP-2738)

## Description


This PR adds the interop country list to the submission of keys screen.
How to test:

When getting a positive diagnosis inside your app, click  'Next' on the diagnosis card. Click 'Next' once more on the Test Result page. Choose your symptoms and click 'Next' once more.
On the "Warning Others" screen if you scroll down, you should see the new country list.
